crates/parry3d/benches/bounding_volume/mod.rs

```diff
@@ -143,7 +142,7 @@ bench_method!(
     bounding_sphere: BoundingSphere,
     c: ConvexHull,
     m: Isometry3<f32>
-);
+);*/
```

Comment:
`ConvexHull` is no longer a type as iitself, but rather an alias to `Vec<Point<Real>>`, I think it would have value to be its own type, but slightly out of scope for this PR ; I'd appreciate instructions as how to proceed:

Should I straight up remove that code, or keep it in and create a tracking issue (+ comment) ?

Then, should I either:
- make it work with the alias ? 
  - implement the methods for `Vec<Point<Real>>`, which will probably be confusing
- make a `ConvexHull` type. An opportunity for better docs in my opinion, `ConvexHull` exists in the current documentation at a few places ([here](i.e: https://github.com/Vrixyz/parry/blob/7e76ebe0c7a155e44d631d4f25a227cf7daf598c/src/query/closest_points/closest_points_support_map_support_map.rs#L8) or [there](https://github.com/Vrixyz/parry/blob/7e76ebe0c7a155e44d631d4f25a227cf7daf598c/src/query/intersection_test/intersection_test_halfspace_support_map.rs#L16-L17)...)

Reply:
The `ConvexHull` shpe type has been renamed `ConvexPolyhedron` in 3D and `ConvexPolygon` in 2D.